docker-swarm.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.1
+    volumes:
+      - ./elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro
```

Comment:
I believe that this will work on a single node cluster but will probably fail on a cluster with nodes > 1.

A solution here would be to define the equivalent of the configuration file as environment variables.